public/index.html

```diff
@@ -23,6 +23,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Galdeano&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
I would recommend you to upload fonts and not use the google link option anymore. [See this article to learn more about the why.](https://www.heise.de/news/DSGVO-Abmahnwelle-wegen-Google-Fonts-7206364.html) :)